DOCKER/Dockerfile.develop

```diff
@@ -0,0 +1,35 @@
+FROM alpine:3.5
+
+ENV DATA_ROOT /tendermint
+ENV TMROOT $DATA_ROOT
+
+RUN addgroup tmuser && \
+    adduser -S -G tmuser tmuser
+
+RUN mkdir -p $DATA_ROOT && \
+    chown -R tmuser:tmuser $DATA_ROOT
+
+RUN apk add --no-cache bash curl jq
+
+ENV GOPATH /go
+ENV PATH "$PATH:/go/bin"
+RUN mkdir -p /go/src/github.com/tendermint/tendermint && \
```

Comment:
It's a compromise between image size and caching. I've designed this so our k8s cluster starts as fast as possible (the smaller the image size, the better). On the other hand, is caching with faster build times, but resulting image size ~ 200Mb (versus 70Mb)